std_msgs/Header header
geometry_msgs/Point[4] poses